Core/Settings/APIv2Settings.cs

```diff
@@ -161,5 +164,10 @@ public static string GetOrderBookKey(this CacheSettings settings, string assetPa
             return string.Format(settings.OrderBooksCacheKeyPattern, assetPairId, isBuy);
         }
     }
+
+    public class ClientAccountRecoverySettings
+    {
+        public long SelfieImageMaxSizeMBytes { get; set; }

```

Comment:
Maybe it is better to make the property optional with a default value = 64Mb

Reply:
Yes it seems that namespace is correct. As we discussed to use enums directly from client library to bypass redundant mappings.
![image](https://user-images.githubusercontent.com/12168520/43839018-1362fc70-9b2e-11e8-99bf-a4933049539a.png)

Reply:
Done, default is 32Mb.